src/changeStatus.js

```diff
@@ -0,0 +1,11 @@
+const changeStatus = (arr, itemId) => {
+  const item = document.getElementById(itemId);
+  arr[itemId].completed = !arr[itemId].completed;
+  if (arr[itemId].completed) {
+    item.classList.add('disabled');
+  } else {
+    item.classList.remove('disabled');
+  }
+};
```

Comment:
- [ ] I could see you implement the change status function, and I'm able to check/uncheck the elements, one important detail here is to use `localStorage` to save the new info and show it when I refresh the page. At this point when I check/uncheck an element the application doesn't save the change. This is required for drag and drop too. 🐶

Reply:
* done 👍